sale_order_line_packaging_qty/views/sale_order.xml

```diff
@@ -38,9 +38,10 @@
                     name="product_packaging"
                     attrs="{'invisible': [('product_id', '=', False)]}"
                     context="{'default_product_id': product_id, 'tree_view_ref':'product.product_packaging_tree_view', 'form_view_ref':'product.product_packaging_form_view'}"
-                    domain="[('product_id','=',product_id)]"
+                    domain="[('product_id','=',product_id), ('qty', '>', 0)]"
```

Comment:
In which case do you have packaging == 0?

Reply:
Not clear how, but thanks for asking the why ? After internal discussion this is not needed any more. And it would be better to fix the data model with a constraint not allowing zero quantity on the product.packaging.